plugins/plugins-api/src/main/java/org/codice/compliance/saml/plugin/IdpPostResponse.java

```diff
@@ -133,22 +133,19 @@ private void parseAndSetFormValues(Node responseForm) {
      *
```

Comment:
@bakejeyner do we still need this comment ☝️ since we're not dealing with hidden fields anymore?

Reply:
Correct.

Reply:
Good catch! We do not need that comment anymore